Makefile.defs

```diff
@@ -192,8 +192,8 @@ GO_CLEAN = $(GO) clean $(GO_CLEAN_FLAGS)
 GO_VET = $(GO) vet
 GO_LIST = $(GO) list
 
-HELM_TOOLBOX_VERSION ?= "v1.0.2"
-HELM_TOOLBOX_SHA ?= "c5a728e61e1f04580c1a0074f9bf3ab96f7beca687385878b403bb0e52163d34"
+HELM_TOOLBOX_VERSION ?= "v1.1.0"
+HELM_TOOLBOX_SHA ?= "961693f182b9b456ed90e5274ac5df81e4af4343104e252666959cdf9570ce9e"
```

Comment:
Heads up I just pushed `v1.1.1` to fix the CVEs in the images. Feel free to either bump it in this PR or I'll bump this later in another PR. https://quay.io/repository/cilium/helm-toolbox/manifest/sha256:722901c26a9edb75a514927f9f759302a0dc03edacadc835a76bf620eb6da66a

Reply:
It looks like `m2r2` is not working in the new image

```
$ make -C Documentation update-helm-values
make: Entering directory '/home/gerald/code/work/cilium-upstream/Documentation'
docker container run --rm --workdir /src/Documentation --volume /home/gerald/code/work/cilium-upstream/Documentation/..:/src --env READTHEDOCS_VERSION= --env SKIP_LINT= --env INCREMENTAL= --user "1000:1000" "quay.io/cilium/helm-toolbox:"v1.1.1"@sha256:"722901c26a9edb75a514927f9f759302a0dc03edacadc835a76bf620eb6da66a"" helm-docs -d -c /src/install/kubernetes -t /src/Documentation/helm-values.tmp.tmpl > helm-values.tmp
time="2024-03-07T06:49:24Z" level=info msg="Found Chart directories [cilium]"
time="2024-03-07T06:49:24Z" level=info msg="Generating README Documentation for chart /src/install/kubernetes/cilium"
awk -F'|' '{print "|"$2"|"$5"|"$3"|"$4"|"}' helm-values.tmp > helm-values.awk
docker container run --rm --workdir /src/Documentation --volume /home/gerald/code/work/cilium-upstream/Documentation/..:/src --env READTHEDOCS_VERSION= --env SKIP_LINT= --env INCREMENTAL= --user "1000:1000" "quay.io/cilium/helm-toolbox:"v1.1.1"@sha256:"722901c26a9edb75a514927f9f759302a0dc03edacadc835a76bf620eb6da66a"" python3 /usr/bin/m2r2 --overwrite helm-values.awk
Traceback (most recent call last):
  File "/usr/bin/m2r2", line 5, in <module>
    from m2r2 import main
ModuleNotFoundError: No module named 'm2r2'
make: *** [Makefile:106: helm-values.rst] Error 1
make: Leaving directory '/home/gerald/code/work/cilium-upstream/Documentation'
```